airflow/gcp/operators/gcs.py

```diff
@@ -625,3 +625,36 @@ def execute(self, context: Dict):
                 object_name=self.destination_object,
                 filename=destination_file.name
             )
+
+
+class GCSDeleteBucketOperator(BaseOperator):
+    """
+    Deletes bucket from a Google Cloud Storage, either
+
+    :param bucket_name: The GCS bucket to delete from
+    :type bucket_name: str
+    """
+
+    template_fields = ('bucket_name', )
+
+    @apply_defaults
+    def __init__(self,
+                 bucket_name: str,
+                 gcp_conn_id: str = 'google_cloud_default',
```

Comment:
Missing docstring

Reply:
I thought this is docstring

`"""
    Deletes bucket from a Google Cloud Storage

    .. seealso::
        For more information on how to use this operator, take a look at the guide:
        :ref:`howto/operator:GCSDeleteBucketOperator


    :param bucket_name: name of the bucket which will be deleted
    :type bucket_name: str
    """`
Isn't it?